src/components/landing/LandingContent.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { TextData } from '../../api/texts';
+
+const tokenImage = require('../../graphics/landing_token.svg');
+const mobileImage = require('../../graphics/landing_phone.png');
+const labelValues = [];
+
+class LandingContent extends React.Component {
+  public props: {
+    text: TextData;
+  };
+
+  constructor(props: any) {
+    super(props);
+    let header1 = this.props.text.value.find((text: any) => {
+      return text.label === 'header1';
+    });
+    let header2 = this.props.text.value.find((text: any) => {
+      return text.label === 'header2';
+    });
+    let header3 = this.props.text.value.find((text: any) => {
+      return text.label === 'header3';
+    });
+    let description = this.props.text.value.find((text: any) => {
+      return text.label === 'description';
+    });
+    let joinBtnTxt = this.props.text.value.find((text: any) => {
+      return text.label === 'joinBtnTxt';
+    });
+    labelValues.push(header1, header2, header3, description, joinBtnTxt);
+  }
+
+  public render() {
+    return (
+      <div className="landing">
+        <img className="token-bg" src={tokenImage} />
+        <div className="container landing-container">
+          <div className="row landing-container-row">
+            <div className="col-sm-6 col-md-6 landing-header-container">
+              <div className="landing-header">
+                <p>{labelValues[0] ? labelValues[0].value : ''}</p>
```

Comment:
This is not very coder friendly.
Why dont use json structure?
Destructering and structering.

You have already labeled them in the construction method, then you unlabel them?

Reply:
I did went to a different approach for this as well.